src/components/nodes/node-sul-library.tsx

```diff
@@ -1,67 +1,188 @@
+                        <div className="su-relative su-flex su-flex-row su-items-start su-mt-20 md:su-mt-18 su-mb-4 su-type-1">
+                          <EnvelopeIcon width={19} className="md:su-absolute md:su-left-[-38px] md:su-top-02em su-mt-01em md:su-mt-0 su-mr-12"/>
+                          <Link className="su-no-underline hocus:su-underline" href={`mailto:${node.su_library__email}`}>
+                            {node.su_library__email}
+                          </Link>
+                        </div>
+                      }
+                      {node.su_library__address &&
+                        <div className="su-relative su-flex su-flex-row su-items-start su-mt-20 md:su-mt-18 su-mb-4 su-type-1">
+                          <MapPinIcon width={19} className="md:su-absolute md:su-left-[-38px] md:su-top-01em su-mt-01em md:su-mt-0 su-mr-12"/>
+                          {node.su_library__map_link ? (
+                            <Link href={node.su_library__map_link.uri} className="su-no-underline hocus:su-underline">
+                              <div>{node.su_library__address.address_line1}</div>
+                              <div>{node.su_library__address.address_line2}</div>
+                              <div>{node.su_library__address.locality}, {node.su_library__address.administrative_area} {node.su_library__address.postal_code}</div>
+                            </Link>
+                          ): (
+                            <>
+                              <div>{node.su_library__address.address_line1}</div>
+                              <div>{node.su_library__address.address_line2}</div>
+                              <div>{node.su_library__address.locality}, {node.su_library__address.administrative_area} {node.su_library__address.postal_code}</div>
+                            </>
+                          )}
+                        </div>
+                      }
+                      {hours &&
+                        <div className="su-relative su-flex su-flex-row su-items-start su-mt-20 md:su-mt-18 su-mb-4 su-type-1">
+                          <ClockIcon width={19} className="md:su-absolute md:su-left-[-38px] md:su-top-01em su-mr-12"/>
+                          {isOpen ? 'Open' : 'Closed'}
+                        </div>
+                      }
+                    </div>
+                    {libraryPrimaryHours &&
+                      <div className="su-relative su-pb-70 md:su-pb-40">
+                        <label htmlFor="library-hours" className="su-sr-only">Choose a library</label>
+                        <select id="library-hours" className="su-absolute su-leading-none su-w-full su-text-black su-py-20 su-mb-20 su-rounded su-shadow su-type-1">
+                          {libraryPrimaryHours.map(date =>
+                              <option key={`library-hours-${date.day}`} selected={date.day === currentDay} disabled={date.day !== currentDay} value={date.day}>
```

Comment:
I'm not sure if they want the options disabled or not. We'll check with them upon review.